chapters/002-smol-internet/README.md

```diff
@@ -608,14 +495,7 @@ root@boudi:/# cat /proc/sys/net/ipv4/ip_forward
 1
 ```
 
-It looks like docker, by default, sets the value on every container to `1`,
-which means, "yeah, forward those packets!" Let's change that value to 0 and see
-what happens! There's a lot of permission shenanigans happening with docker...
-So, in order to turn off packet forwarding on `boudi`, we need to change our
-docker-compose.yml file. docker-compose exposes `sysctls` which allows us to
-change default kernel settings. We have explicitly added that setting to `boudi`
-and it should currently be set to `1`. Change it to `0` to disable ip
-forwarding.
+It looks like docker, by default, sets the value on every container to `1`, which means, "yeah, forward those packets!" Let's change that value to 0 and see what happens! There's a lot of permission shenanigans happening with docker...  So, in order to turn off packet forwarding on `boudi`, we need to change our docker-compose.yml file. docker-compose exposes `sysctls` which allows us to change default kernel settings. We have explicitly added that setting to `boudi` and it should currently be set to `1`. Change it to `0` to disable ip forwarding.
```

Comment:
Again... maybe we add this to the docker-compose by default and then muddle with it? I think we would still have to offer explanation and I don't know that it would be any cleaner to explain what's in the docker-compose and then have people go back and edit it after the fact?